agogosml/agogosml/common/flask_http_listener_client.py

```diff
@@ -46,6 +49,7 @@ def start(self, on_message_received):
         self.on_message_received = on_message_received
         self.t_flask = threading.Thread(name='agogosml', target=self.run_flask_server)
         self.t_flask.setDaemon(True)
+        self.logger.event('Starting Flask Server on Port {0}, and Host {1}'.format(self.port, self.host))
```

Comment:
The event should likely be a small easily seachable string like 'flask.server.start'. You can pass the parameters via props, like this:

```py
self.logger.event('flask.server.start', {'port': str(self.port), 'host': self.host})
```